### ISSUE TITLE
Can not dump unprivileged lxc container

### ISSUE THREAD

**Issue by chenhaiq** [xemul/criu#151]
It hits this error:
```
(00.953319) Dump NET namespace info 9 via 9713
(00.953476) Mount ns' sysfs in crtools-sys.yZWolu
(00.953490) Error (net.c:1087): mount failed: Device or resource busy
(00.967029) Error (namespaces.c:910): Namespaces dumping finished with error 65280
```

lxc conf:
```
lxc.include = /usr/share/lxc/config/ubuntu.common.conf
lxc.include = /usr/share/lxc/config/ubuntu.userns.conf
lxc.arch = x86_64

# Container specific configuration
lxc.aa_allow_incomplete = 1
lxc.console = none
lxc.tty = 0
lxc.cgroup.devices.deny = c 5:1 rwm
lxc.seccomp =
lxc.mount.auto = cgroup:mixed
lxc.autodev = 1
lxc.kmsg = 0
lxc.include = /usr/share/lxc/config/common.conf.d/00-lxcfs.conf
lxc.id_map = u 0 100000 65536
lxc.id_map = g 0 100000 65536
lxc.rootfs = /var/lib/lxc/w7/rootfs
lxc.rootfs.backend = dir
lxc.utsname = w7

# Network configuration
lxc.network.type = veth
lxc.network.link = lxcbr0
lxc.network.flags = up
lxc.network.hwaddr = 00:16:3e:b0:d1:fc

```

**Comment by chenhaiq** [comment]
BTW, gdb break point does not work for "mount_ns_sysfs". It won't break witth "break mount_ns_sysfs". Why?

**Comment by tych0** [comment]
There was a bug in older xenial kernels. Try Kernel 4.4.0-21.

**Comment by chenhaiq** [comment]
@tych0 yes, this bug is fixed in kernel 4.4.0-21.

**Comment by tych0** [comment]
On Fri, Apr 22, 2016 at 07:55:47AM -0700, Chen Hai Quan wrote:
> @tych0 yes, this bug is fixed in kernel 4.4.0-21.

Cool, glad to hear it.
